test/FunctionalTests/Microsoft.OData.Client.Tests/DataServiceContextTests.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Xunit;
+
+namespace Microsoft.OData.Client.Tests
+{
+    public class DataServiceContextTests
+    {
+        [Fact]
+        public void KeyComparisonGeneratesFilterQuery_Defaults_To_True()

```

Comment:
You couldn't find a home for this 1 test where other tests for `DataServiceContext` are domiciled?

Reply:
The other `DataServiceContext`-related tests are in the E2E tests. I thought it would be good to have some coverage in the unit tests as well but didn't find a relevant class, so I created one.